collect_app/src/main/res/layout/project_settings_dialog_layout.xml

```diff
@@ -47,4 +50,28 @@
         app:layout_constraintStart_toStartOf="@+id/general_settings_button"
         app:layout_constraintTop_toBottomOf="@+id/general_settings_button" />
 
+    <View
+        android:id="@+id/divider"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_marginTop="@dimen/margin_standard"
+        android:background="?colorPrimary"
+        app:layout_constraintStart_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/admin_settings_button" />
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/about_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_standard"
+        android:layout_marginTop="@dimen/margin_standard"
+        android:layout_marginBottom="@dimen/margin_standard"
+        android:background="?attr/selectableItemBackgroundBorderless"
+        android:gravity="center_vertical"
+        android:text="@string/about_preferences"
+        app:icon="@drawable/ic_outline_info_24"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/divider"/>
```

Comment:
It feels like so much work to create a basic linear flow with `ConstraintLayout`. It does seem like they don't want us using `LinearLayout` though: https://developer.android.com/guide/topics/ui/layout/linear. There seems to be some new "Flow" component but it's hard to find good docs on using it.

We can have a play this when we chat later.